lib/algolia.rb

```diff
@@ -36,6 +36,7 @@
 require 'algolia/error'
 require 'algolia/search_index'
 require 'algolia/logger_helper'
+require 'algolia/webmock'
```

Comment:
I we add this here, we require it by default every time, instead of letting the user decide when to include it 🙂 
```suggestion
```